playbooks/roles/cluster/files/openshift_tower_auth.yml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
Same as previous comment, should this be a template? Also, could we have matching names for these vars? Only difference seems to 4 additional chars i.e. `username` and `password`

Reply:
The difference in the vars is because I'm overwriting the default vars of the openshift authenticate job. and it's not a template because I'm supplying it as a file to the the created job_template as the jobs extra vars.

Again, i'm not against writing it out locally, we're just avoiding having plain text credentials on the box this way.